sdk/core/azure-core/vcpkg/portfile.cmake

```diff
@@ -1,6 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
+# NOTE: All changes made to this file will get overwritten by the next port release.
```

Comment:
nit: Any, instead of All

```suggestion
# NOTE: Any changes made to this file will get overwritten by the next port release.
```

Reply:
nit: Any change (singular).
We can talk more here/offline, but I think "all" is more accurate - "changes may accumulate but then in one moment all of them will be lost - not one, not a few of them at random, but simply all of them - the whole sequential range from first to the last one".